.gitignore

```diff
@@ -6,3 +6,6 @@ bin/
 
 # Mac files
 .DS_Store
+
+# VS Code setting 
+.vscode
```

Comment:
Can you remove this. This is very user specific setting and shouldn't be added to project gitignore. You should add it to you global gitignore.

Reply:
Isn't adding IDE config to gitignore one of the usage of gitignore design? That's helpful for devs to automatically avoid committing local dev config to project

Here is a guidance of what we should add to gitignore: https://www.atlassian.com/git/tutorials/saving-changes/gitignore
 ```
dependency caches, such as the contents of /node_modules or /packages
compiled code, such as .o, .pyc, and .class files
build output directories, such as /bin, /out, or /target
files generated at runtime, such as .log, .lock, or .tmp
hidden system files, such as .DS_Store or Thumbs.db
personal IDE config files, such as .idea/workspace.xml
```

I notice that our existing gitignore also includes the Intellij config: https://github.com/coinbase/rosetta-cli/blob/master/.gitignore#L5

Reply:
Projects should add project specific gitignore not users setups specific gitignore. There's so many IDEs that we can't really add all to gitignore. That's why global user specific gitignore exists. I think those existing IDE specific gitignore should also go away

Reply:
https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files

It's easier to have your IDE specific global gitignore for all your projects rather than adding them to project specific gitignore.

Reply:
Okay, I can remove the existing IDE config to make the gitignore clean